doc/src/manual/ranch.asciidoc

```diff
@@ -88,6 +88,13 @@ Unique name used to refer to a listener.
 [source,erlang]
 ----
 transport_opts(SocketOpts) = #{
+    alarms               => #{}
+                          | #{
+                                type := num_connections,
+                                treshold := non_neg_integer(),
+                                callback := fun((ref(), term(), pid(), [pid()]) -> any()),
+                                cooldown => non_neg_integer()
+                            },
```

Comment:
That's wrong, the type is `#{term() => #{ type := ... }}`.

There's also no need to have `#{}` there separately.

Reply:
Fixed.